api-description/forms/pk/post.json

```diff
@@ -0,0 +1,23 @@
+{
+    "description": "Endpoint to answer a form",
+    "url": "http://localhost:8000/forms/1",
+    "fields": {
+        "answers": {
+            "1": "Friends",
+            "2": "one_time",
+            "3": ["two_times", "three_times"],
+            "4": 5
+        }
+    },
+    "OK": {
+        "status": 201,
+        "response_data": {
+            "answers": {
+                "1": "Friends",
+                "2": "one_time",
+                "3": ["two_times", "three_times"],
+                "4": 4
```

Comment:
Actually it might be `"5"` in practice.  I think so.  I'm not sure, depends on the internals of the process.  Will fix it if it turns out to be so.